src/model.rs

```diff
@@ -130,7 +144,7 @@ impl NodeState {
     }
 }
 
-#[derive(Default, Serialize)]
+#[derive(Default)]
```

Comment:
I've removed `Serialize` here because `Instant` doesn't trivially implement it. It wasn't used anywhere so I'd be curious if this is necessary for consumption of the API?

Since it's not trivial to just convert an Instant to seconds or something like that (at least with STD) that would likely make things at least a tiny bit more complicated. Worst-case scenario one might have to pass in something like a startup time stored on the `ScuttleButt`, which doesn't need Serialize, that would definitely work. But maybe not necessary?

Reply:
This last_heartbeat thing is not gossiped and lives outside of the Scuttlebutt state. It does not need to be serialized.